### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Bombaleo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# test
```
Comment (DmitryAvramec) [inline README.md]: Hi there!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!11111

**Comment by Bombaleo** [comment]
I fix this shit

**Comment by andersann** [comment]
WTF?

**Comment by Bombaleo** [comment]
=)

**Comment by Bombaleo** [comment]
fuuucj

**Comment by andersann** [comment]
I'm here too

**Comment by Bombaleo** [comment]
dich

**Comment by andersann** [comment]
Awesome commits :100:

**Pull request closed without merging**